apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/queries.ex

```diff
@@ -450,6 +451,53 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Queries do
     end
   end
 
+  def fetch_device_capabilities(db_client, device_id) do
+    device_capabilities_statement = """
+    SELECT purge_properties_compression_format
+    FROM devices
+    WHERE device_id=:device_id
+    """
+
+    device_capabilities_query =
+      DatabaseQuery.new()
+      |> DatabaseQuery.statement(device_capabilities_statement)
+      |> DatabaseQuery.put(:device_id, device_id)
```

Comment:
We're struggling to move away from `CQEx` because it's an [unmaintained library](https://github.com/cqerl/cqex) (and just wraps cqerl, with its strange errors). For new queries, it would be better to use the `Xandra` library, as it's being done e.g. in [line 948](https://github.com/astarte-platform/astarte/blob/7e10975abf83518f4ae7de52e1a6150bdf404f9a/apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/queries.ex#L948) and following